command/sub8_launch/launch/subsystems/cameras/launch/pt_grey_stereo.launch

```diff
@@ -0,0 +1,95 @@
+      <remap from="image_raw" to="left/image_raw" />
+    </node>
+  
+    <!-- mono rectification -->
+    <node pkg="nodelet" type="nodelet" name="left_image_proc_rect"
+         args="load image_proc/rectify camera_nodelet_manager">
+      <remap from="image_mono" to="left/image_mono" />
+      <remap from="image_rect" to="left/image_rect" />
+    </node>
+  
+    <!-- color rectification -->
+    <node pkg="nodelet" type="nodelet" name="left_image_proc_rect_color"
+         args="load image_proc/rectify camera_nodelet_manager">
+      <remap from="image_mono" to="left/image_color" />
+      <remap from="image_rect" to="left/image_rect_color" />
+    </node>
+  
+    <!-- RIGHT CAMERA -->
+    <node pkg="nodelet" type="nodelet" name="right_camera_nodelet"
+        args="load pointgrey_camera_driver/PointGreyCameraNodelet camera_nodelet_manager" >
+      <param name="frame_id" value="/stereo/right_cam" />
+      <param name="serial" value="15356350" />
+      <param name="guid" value="00b09d0100ea51be"/>
+      <param name="camera_info_url" value="file://$(find sub8_launch)/launch/subsystems/cameras/calibration/00b09d0100ea51be.yaml" />
+      <param name="num_dma_buffers" value="10"/>
+      <param name="video_mode" value="format7_mode1"/>
+      <param name="format7_packet_size" value="1200"/>
+      <param name="format7_color_coding" value="raw8"/>
+      <param name="bayer_method" value=""/>
+      <param name="binning_x" value="1"/>
+      <param name="binning_y" value="1"/>
+      <param name="auto_white_balance" value="2"/>
+      <param name="use_ros_time" value="true"/>
+      <remap from="camera" to="right"/>
+      <remap from="image_raw" to="right/image_raw"/>
+      <remap from="set_camera_info" to="right/set_camera_info"/>
+    </node>
+  
+    <!-- Bayer color decoding -->
+    <node pkg="nodelet" type="nodelet" name="right_image_proc_debayer"
+          args="load image_proc/debayer camera_nodelet_manager">
+      <remap from="image_color" to="right/image_color" />
+      <remap from="image_mono" to="right/image_mono" />
+      <remap from="image_raw" to="right/image_raw" />
+    </node>
+  
+    <!-- mono rectification -->
+    <node pkg="nodelet" type="nodelet" name="right_image_proc_rect"
+         args="load image_proc/rectify camera_nodelet_manager">
+      <remap from="image_mono" to="right/image_mono" />
+      <remap from="image_rect" to="right/image_rect" />
+    </node>
+  
+    <!-- color rectification -->
+    <node pkg="nodelet" type="nodelet" name="right_image_proc_rect_color"
+         args="load image_proc/rectify camera_nodelet_manager">
+      <remap from="image_mono" to="right/image_color" />
+      <remap from="image_rect" to="right/image_rect_color" />
+    </node>
+  
+    <!-- Stereo Image Processing -->
+    <node pkg="stereo_image_proc" type="stereo_image_proc" respawn="false" name="stereo_image_proc" args="_approximate_sync:=True"/>
+  </group>
+</launch>
```

Comment:
Yeah, I'll make do. This wasn't too bad, nothing like the camera hell @rleyva had to deal with last year.